app/views/ConfirmationPageView.scala.html

```diff
@@ -72,6 +72,7 @@
     )
 
     @p(link(linkMessage = messages(s"cf.historic.document.request.confirmation.${fileRole.name}.link-text"),
```

Comment:
I know you didn't do it but we can remove the `@p` because semantically wrapping a P tag in a P tag is not correct.
```suggestion
   @link(linkMessage = messages(s"cf.historic.document.request.confirmation.${fileRole.name}.link-text"),
               location = returnLink,
               pClass="govuk-body govuk-!-margin-bottom-9"
            )
```